checklists/newHire.json

```diff
@@ -246,5 +246,11 @@
 		"description": "<a href=\"https://handbook.18f.gov/travel-101/#travel-card\">Instructions</a>",
 		"daysToComplete": 30,
 		"dependsOn": ["travelTraining"]
+	},
+	"filloutsurvey": {
+		"displayName": "Fill out onboarding survey",
+		"description": "<a href=\"https://goo.gl/forms/mBe4bxLydYHZ9wAe2\">Survey<a/a>",
```

Comment:
At the end of `description` the closing anchor tag has a typo:

`<a/a>` should be `</a>`